server/views/pages/pop/appointment-type.njk

```diff
@@ -0,0 +1,41 @@
+{% extends "../../partials/layout.njk" %}
```

Comment:
This needs to come out for this view appointments feature - move to another future branch maybe?